### PR TITLE
Added function references and example

### PR DESCRIPTION
**Pull request by JediFodder**
Added a function reference for the sound and the tcs3472 libraries 
Added an example for turning the lights on and off

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,36 @@ Your class instance, `bme`, will now have the following methods:
...
+```
+import sound
+sound = sound.sound()
+```
+
+Your class instance, `sound`, will now have the following methods:
+
+* `sound.read()` - take a reading of the sound level which you can use to compare different sound levels 
+* `sound.wait_for_double_clap()` - listen for a high sound level twice in a second
+* `sound.wait_for_clap()` - listen for a high sound level once in a second
```
Comment (Gadgetoid) [inline README.md]: Should probably clarify what this does if a high sound level is detected. IE: the return value.

File: README.md [hunk 2]
```diff
@@ -42,3 +42,36 @@ Your class instance, `bme`, will now have the following methods:
...
+```
+import tcs3472
+
+light_sensor = tcs3472.tcs3472() 
+```
+
+Your class instance, `light_sensor`, will now have the following methods:
+
+* `light_sensor.rgb()` - returns the corrected levels of red,green and blue out of 255  
+* `light_sensor.scaled()` - return the amounts of red,green and blue on a scale of 0-1
```
Comment (Gadgetoid) [inline README.md]: Pedantry, but add a space after the commas

Comment (JediFodder) [reply README.md]: added space

File: README.md (hunk 1 above)
Comment (JediFodder) [reply README.md]: Clarified the reference

**Comment by Gadgetoid** [comment]
Thank you!

**Comment by JediFodder** [comment]
Thank you Phil really appreciate it.

**Pull request merged**